css/properties/offset-position.json

```diff
@@ -0,0 +1,55 @@
+{
+  "css": {
+    "properties": {
+      "offset-position": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/offset-position",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "Enabled by default in Chrome Canary only."
```

Comment:
I have a few issues with this. It wouldn't likely just be in desktop. I'd assume it is in Android and WebView. But I can't tell you because it's in Canary. Also, 'Canary' has a short lifespan since it changes every six weeks.

Reply:
Perhaps we should leave Chrome as just `false` and not worry about a note, then?

Reply:
I'd be happy with that resolution. How about you @jpmedley ?

Reply:
Sounds good.